src/app/client/src/app/modules/dashboard/components/report/report.component.ts

```diff
@@ -108,7 +109,7 @@ export class ReportComponent implements OnInit {
       })
     );
 
-
+    this.reportSummaryLabel = `Add ${_.get(this.resourceService, 'frmelmnts.lbl.reportSummary')}`;
```

Comment:
you don't need explicity a new variable for this ..
check this condition inside the template based on currentReportSummary variable... 
currentReportSummary ? Update Report Summary : Add Report Summary

Reply:
The same review goes for the code block for opening of chart & report summary modal .. Please update it there as well...

Reply:
done

Reply:
resolved